Competition/src/main/cpp/subsystems/Feeder.cpp

```diff
@@ -32,19 +30,20 @@ void Feeder::init()
     motor_intake.SetIdleMode(rev::CANSparkMax::IdleMode::kCoast);
     motor_intake.SetInverted(false);
 
-    motor_stage1.RestoreFactoryDefaults();
-    motor_stage1.SetIdleMode(rev::CANSparkMax::IdleMode::kCoast);
-    motor_stage1.SetInverted(false);
-
-    motor_stage2.RestoreFactoryDefaults();
-    motor_stage2.SetIdleMode(rev::CANSparkMax::IdleMode::kCoast);
-    motor_stage2.SetInverted(false);
+    motor_stage.RestoreFactoryDefaults();
+    motor_stage.SetIdleMode(rev::CANSparkMax::IdleMode::kBrake);
+    motor_stage.SetInverted(false);
 
     table->PutBoolean("Reverse Feeder?", false);
-    table->PutNumber("Intake Reverse Speed", FeederConstants::DEFUALT_INTAKE_SPEED_REVERSE);
-    table->PutNumber("Feeder Reverse Speed", FeederConstants::DEFUALT_FEEDER_SPEED_REVERSE);
-    table->PutNumber("Intake Forward Speed", FeederConstants::DEFUALT_INTAKE_SPEED_FORWARD);
-    table->PutNumber("Feeder Forward Speed", FeederConstants::DEFUALT_FEEDER_SPEED_FORWARD);
+    table->PutNumber("Intake Reverse Speed", FeederConstants::DEFAULT_INTAKE_SPEED_REVERSE);
+    table->PutNumber("Feeder Reverse Speed", FeederConstants::DEFAULT_FEEDER_SPEED_REVERSE);
+    table->PutNumber("Intake Forward Speed", FeederConstants::DEFAULT_INTAKE_SPEED_FORWARD);
+    table->PutNumber("Feeder Forward Speed Default", FeederConstants::DEFAULT_FEEDER_SPEED_FORWARD_DEFAULT);
+    table->PutNumber("Feeder Forward Speed Shoot", FeederConstants::DEFAULT_FEEDER_SPEED_FORWARD_SHOOT);
+    
+    state.spiked = false;
```

Comment:
Call resetState here instead to make sure logic is consistent

Reply:
replaced the 3 manual resets with 1 call to reset state